packages/opentelemetry-host-metrics/src/BaseMetrics.ts

```diff
@@ -33,7 +33,7 @@ export interface MetricsCollectorConfig {
   url?: string;
 }
 
-const DEFAULT_NAME = 'opentelemetry-host-metrics';
+const DEFAULT_NAME = '@opentelemetry/host-metrics';
```

Comment:
IIUC the changelog will automatically get this PR title added to it for the next release. However, 

1. that doesn't include the explicit strings, so won't be as helpful as possible.
2. Do you think I should add the bang (`feat(host-metrics)!: ...`) so that this gets added to the "breaking change" section and triggers a new major version (`0.34.0`)?

Reply:
Yeah, I think 0.34.0 should be used for next release.